cypress/integration/account.test.ts

```diff
@@ -50,7 +50,7 @@ describe("Account page", () => {
 
     cy.fixture("guy.jpg", null).as("image");
     cy.get("input[type=file]").selectFile("@image", { force: true });
-
+    cy.wait(500);
```

Comment:
Is there a way to do this using a tag, like I did for `cy.wait("@findAlgorithmsTextbook")`? It would make the test results more reliable, and would allow this step to run faster than 500ms if the machine is powerful enough.

Reply:
I was struggling to find a good way to do this (it was very finicky) but I'll give it another shot